docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,317 @@
+fastLaneLengthSlots = 10
+```
+
+## GateSeal
+
+```python
+seal_duration = 518400  # 6 days as seconds (2 x gov time)
+expiry_timestamp = 1714521600  # 2024-05-01 00:00 UTC
+```
+
+## Roles setup
+
+### **Lido**
+*Aragon app*
+
+**New**
+- `UNSAFE_CHANGE_DEPOSITED_VALIDATORS_ROLE`
+    - None
+
+**Kept**
+- `PAUSE_ROLE`
+    - Voting
+- `RESUME_ROLE`
+    - Voting
+- `STAKING_PAUSE_ROLE`
+    - Voting
+- `STAKING_CONTROL_ROLE`
+    - Voting
+
+**Obsolete**
+- `MANAGE_FEE` `[MN, GL]`
+- `MANAGE_WITHDRAWAL_KEY` `[MN, GL]`
+- `MANAGE_PROTOCOL_CONTRACTS_ROLE` `[MN, GL]`
+- `BURN_ROLE` `[MN, GL]`
+- `DEPOSIT_ROLE` `[MN, GL]`
+- `SET_EL_REWARDS_VAULT_ROLE` `[MN, GL]`
+- `SET_EL_REWARDS_WITHDRAWAL_LIMIT_ROLE` `[MN,GL]`
+
+### **NodeOperatorsRegistry**
+*Aragon app*
+
+**New**
+- `STAKING_ROUTER_ROLE`
+    - StakingRouter (set in voting script)
+- `MANAGE_NODE_OPERATOR_ROLE`
+    - None (maybe Voting once upgrade applied?)
+
+**Kept**
+- `MANAGE_SIGNING_KEYS`
+- `SET_NODE_OPERATOR_LIMIT_ROLE`
+
+**Obsolete**
+- `ADD_NODE_OPERATOR_ROLE` `[MN, GL]`
+- `SET_NODE_OPERATOR_ACTIVE_ROLE` `[MN, GL]`
+- `SET_NODE_OPERATOR_NAME_ROLE` `[MN, GL]`
+- `SET_NODE_OPERATOR_ADDRESS_ROLE` `[MN, GL]`
+- `REPORT_STOPPED_VALIDATORS_ROLE` `[MN, GL]`
+
+### **LegacyOracle (former LidoOracle)**
+*Aragon app*
+
+**New**
+None
+
+**Kept**
+None
+
+**Obsolete**
+- `MANAGE_MEMBERS` `[MN, GL]`
+- `MANAGE_QUORUM` `[MN, GL]`
+- `SET_BEACON_SPEC` `[MN, GL]`
+- `SET_REPORT_BOUNDARIES` `[MN, GL]`
+- `SET_BEACON_REPORT_RECEIVER` `[MN, GL]`
+
+### **AccountingOracle (OZ)**
+*OZ, Proxy*
+
+- PROXY ADMIN
+    - Agent
+- `DEFAULT_ADMIN_ROLE`
+    - Agent
+- `SUBMIT_DATA_ROLE`
+    - None
+- `MANAGE_CONSENSUS_CONTRACT_ROLE`
+    - None
+- `MANAGE_CONSENSUS_VERSION_ROLE`
+    - None
+
+### **Burner**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE`
+    - Agent
+- `REQUEST_BURN_SHARES_ROLE`
+    - granted to Lido in Burner constructor
+    - NodeOperatorsRegistry
+- `REQUEST_BURN_MY_STETH_ROLE`
+    - None
+- `RECOVER_ASSETS_ROLE`
+    - None
+
+### **LidoLocator (OZ)**
+*OZ, Proxy*
+
+- PROXY ADMIN
+    - Agent
+
+### **StakingRouter (OZ)**
+*OZ, Proxy*
+
+- PROXY ADMIN
+    - Agent
+- `DEFAULT_ADMIN_ROLE`
+    - Agent
+- `MANAGE_WITHDRAWAL_CREDENTIALS_ROLE`
+    - None
+- `STAKING_MODULE_PAUSE_ROLE`
+    - DepositSecurityModule
+- `STAKING_MODULE_RESUME_ROLE`
+    - DepositSecurityModule
+- `STAKING_MODULE_MANAGE_ROLE`
+    - None
+- `REPORT_EXITED_VALIDATORS_ROLE`
+    - AccountingOracle
+- `UNSAFE_SET_EXITED_VALIDATORS_ROLE`
+    - None
+- `REPORT_REWARDS_MINTED_ROLE`
+    - Lido
+
+### **HashConsensus for AccountingOracle (OZ)**
+*OZ, No Proxy*
+
+- `DEFAULT_ADMIN_ROLE`
+    - Agent
+- `MANAGE_MEMBERS_AND_QUORUM_ROLE`
+    - None
+- `DISABLE_CONSENSUS_ROLE`
+    - None
+- `MANAGE_FRAME_CONFIG_ROLE`
+    - None
+- `MANAGE_FAST_LANE_CONFIG_ROLE`
+    - None
+- `MANAGE_REPORT_PROCESSOR_ROLE`
+    - None
+- `address[] _memberAddresses`
+    - Current LidoOracle Committee (migrated in UpgradeTemplate)
```

Comment:
do we want to have the same description here and in (`ValidatorExitBusOracle`) for the other values such as frame and chain config?

Reply:
I've added some of them which are specified during the contracts deployment (as constructor args). Others look like out of scope for me bacause they are set automatically or in the template

Reply:
maybe link to source or list will be useful to share, because this doc will be used while verification